api/envoy/extensions/filters/http/aws_request_signing/v3/aws_request_signing.proto

```diff
@@ -19,11 +19,19 @@ option (udpa.annotations.file_status).package_version_status = ACTIVE;
 // [#extension: envoy.filters.http.aws_request_signing]
 
 // Top level configuration for the AWS request signing filter.
-// [#next-free-field: 6]
+// [#next-free-field: 7]
 message AwsRequestSigning {
   option (udpa.annotations.versioning).previous_message_type =
       "envoy.config.filter.http.aws_request_signing.v2alpha.AwsRequestSigning";
 
+  enum SigningAlgorithm {
+    // Use SigV4 for signing
+    AWS_SigV4 = 0;
+
+    // Use SigV4A for signing
+    AWS_SigV4A = 1;
```

Comment:
nit: basically, we prefer all uppercase enum value.

Reply:
ack